Updates/0335_gossip_menu_option_broadcast_text_id.sql

```diff
@@ -0,0 +1,19919 @@
+
+UPDATE gossip_menu_option SET option_broadcast_text = 8261 WHERE id = 1 AND menu_id = 63;
+
+# --------------------------------------------------------------------------------
+# ## tab 64/0	=> "I am interested in mage training."
+# == mal [2522] "I am interested in mage training."
+
+UPDATE gossip_menu_option SET option_broadcast_text = 2522 WHERE id = 0 AND menu_id = 64;
+
+# --------------------------------------------------------------------------------
+# ## tab 64/1	=> "I wish to unlearn my talents."
+# == mal [8261] "I wish to unlearn my talents."
+
+UPDATE gossip_menu_option SET option_broadcast_text = 8261 WHERE id = 1 AND menu_id = 64;
+
+# --------------------------------------------------------------------------------
+# ## tab 83/0	=> "Return me to life."
+# == mal [2547] "Return me to life."
+
+UPDATE gossip_menu_option SET option_broadcast_text = 2547 WHERE id = 0 AND menu_id = 83;
+
+# --------------------------------------------------------------------------------
+# ## tab 85/0	=> "Can you train me how to use rogue skills?"
+# == mal [7491] "Can you train me how to use rogue skills?"
+
+UPDATE gossip_menu_option SET option_broadcast_text = 7491 WHERE id = 0 AND menu_id = 85;
+
+# --------------------------------------------------------------------------------
+# ## tab 85/1	=> "I wish to unlearn my talents."
+# == mal [8261] "I wish to unlearn my talents."
+
+UPDATE gossip_menu_option SET option_broadcast_text = 8261 WHERE id = 1 AND menu_id = 85;
+
+# --------------------------------------------------------------------------------
+# ## tab 125/0	=> "You've got something I need, Astor. And I'll be taking it now."
+# == mal [2591] "You've got something I need, Astor. And I'll be taking it now."
+
+UPDATE gossip_menu_option SET option_broadcast_text = 2591 WHERE id = 0 AND menu_id = 125;
+
+# --------------------------------------------------------------------------------
+# ## tab 126/0	=> "You're Astor Hadren, right?"
+# == mal [2589] "You're Astor Hadren, right?"
+
+UPDATE gossip_menu_option SET option_broadcast_text = 2589 WHERE id = 0 AND menu_id = 126;
+
+# --------------------------------------------------------------------------------
+# ## tab 141/0	=> "Can you train me how to use rogue skills?"
+# == mal [7491] "Can you train me how to use rogue skills?"
+
+UPDATE gossip_menu_option SET option_broadcast_text = 7491 WHERE id = 0 AND menu_id = 141;
+
+# --------------------------------------------------------------------------------
+# ## tab 141/1	=> "I wish to unlearn my talents."
+# == mal [8261] "I wish to unlearn my talents."
+
+UPDATE gossip_menu_option SET option_broadcast_text = 8261 WHERE id = 1 AND menu_id = 141;
+
+# --------------------------------------------------------------------------------
+# ## tab 141/4	=> "<Take the letter>"
+# NONE FOUND
+
+# --------------------------------------------------------------------------------
+# ## tab 161/1	=> "[PH] Get Thieves' Tools"
+# NONE FOUND
+
```

Comment:
According to wotlk-db it should be bid=2621

UPDATE gossip_menu_option SET option_broadcast_text = 2621 WHERE id = 1 AND menu_id = 161;